packages/launchpad/cypress/e2e/scaffold-component-testing.cy.ts

```diff
@@ -83,6 +83,26 @@ describe('scaffolding component testing', {
       cy.findByRole('button', { name: 'Continue' }).click()
       verifyConfigFile(`cypress.config.ts`)
     })
+
+    it('detects react dependency even if `package.json` is not declared in `exports`', () => {
+      cy.scaffoldProject('react-vite-ts-unconfigured')
```

Comment:
We have a create-react-app fixture, should we just use that? Then you won't need to hack into the `package.json` - it already lacks `main` and `exports`.

Reply:
Right, but I wanted to test against a project that has `exports` defined, but doesn't export `package.json` since that's where we run into the issue